apps/src/music/views/PreviewControls.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+const classNames = require('classnames');
+const FontAwesome = require('../../templates/FontAwesome');
+const moduleStyles = require('./preview-controls.module.scss').default;
+
+/**
+ * Set of controls for previewing sounds in various custom Music Lab block fields
+ */
+const PreviewControls: React.FunctionComponent<
+  PreviewButtonProps & ClearButtonProps
+> = props => (
+  <div className={moduleStyles.controlsRow}>
+    <PreviewButton {...props} />
+    <ClearButton {...props} />
+  </div>
+);
+
+interface ClearButtonProps {
+  onClickClear: () => void;
+  cancelPreviews: () => void;
+}
+
+const ClearButton: React.FunctionComponent<ClearButtonProps> = ({
+  onClickClear,
+  cancelPreviews
+}) => (
+  <FontAwesome
+    icon={'trash-o'}
+    onClick={() => {
+      cancelPreviews();
+      onClickClear();
+    }}
+    className={moduleStyles.previewButton}
+  />
+);
+
+interface PreviewButtonProps {
+  enabled: boolean;
+  playPreview: () => void;
+}
+
+const PreviewButton: React.FunctionComponent<PreviewButtonProps> = ({
+  enabled,
+  playPreview
+}) => {
+  return (
+    <FontAwesome
```

Comment:
Also rather general question rather than a specific to this PR at this point.
I'm not sure that using `<i>` as a button is the best option since we're trying to migrate to semantic html in the future.
So the question is: what prevents us from using `apps/src/templates/Button.jsx` here? And what are the desired improvements should be done to that component in order to use it here and anywhere else in the cdo. (Since that Button component is going to be universal at some point in the future, and I guess it'll be me who probably will be updating it)

Reply:
Great call. Using `Button.jsx` or at least `<button>` is the right thing to do here. I'll update this.

Reply:
In the spirit of Music Lab avoiding importing older components, at least for now, https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/templates/Button.jsx does look like it's a bit heavy for what we need.

Reply:
Yeah, the current state of `<Button/>` is far from perfect. However once we move everything to one design system we'll need to not forget to update it as well

Reply:
Wrapped these in `<button>`s for now.